Sources/TuistEnvKit/Commands/TuistCommand.swift

```diff
@@ -22,17 +22,39 @@ public struct TuistCommand: ParsableCommand {
 
     public static func main(_: [String]? = nil) -> Never {
         let errorHandler = ErrorHandler()
+        let processedArguments = processArguments()
+
+        // Help env
+        if processedArguments.dropFirst().first == "--help-env" {
+            let error = UnhandledError(error: CleanExit.helpRequest(self))
+            errorHandler.fatal(error: error)
+            _exit(exitCode(for: error).rawValue)
+        }
```

Comment:
This does not properly handle the `--help-env` argument.

```suggestion
        if processedArguments.dropFirst().first == "--help-env" {
            let error = CleanExit.helpRequest(self)
            exit(withError: error)
        }
```